internal/xds/translator/extauth.go

```diff
@@ -93,7 +93,10 @@ func extAuthFilterName(route *ir.HTTPRoute) string {
 }
 
 func extAuthConfig(extAuth *ir.ExtAuth) *extauthv3.ExtAuthz {
-	config := &extauthv3.ExtAuthz{}
+	config := &extauthv3.ExtAuthz{
+		TransportApiVersion: corev3.ApiVersion_V3,
```

Comment:
why not use `AUTO` here ?

Reply:
It's deprecated. We should set it to v3 explicitly.
```
	// When not specified, we assume v2, to ease migration to Envoy's stable API
	// versioning. If a client does not support v2 (e.g. due to deprecation), this
	// is an invalid value.
	//
	// Deprecated: Marked as deprecated in envoy/config/core/v3/config_source.proto.
	ApiVersion_AUTO ApiVersion = 0
```